erts/emulator/beam/erl_arith.c

```diff
@@ -461,6 +461,70 @@ erts_mixed_plus(Process* p, Eterm arg1, Eterm arg2)
     }
 }
 
+Eterm
+erts_unary_minus(Process* p, Eterm arg)
```

Comment:
This is a copy and paste of erts_mixed_minus, without the first argument, and with zero inlined.